nixos/modules/services/web-servers/keter/bundle.nix

```diff
@@ -0,0 +1,41 @@
+/* This makes a keter bundle as described on the github page:
+  https://github.com/snoyberg/keter#bundling-your-app-for-keter
+
+  Note that we're not actually putting in the app into the bundle,
+  we already can use the nix store for copying, so we just
+  symlink to the app.
+*/
+{ domain
+, executable
+, gnutar
+, writeTextFile
+, lib
+, stdenv
+, ...
+}:
+
+let
+  str.stanzas = [{
+    # we just use nix as an absolute path so we're not bundling any binaries
+    type = "webapp";
```

Comment:
I don't know much about keter, but what if someone wants a different configuration? You should expose this `str` thing. (And maybe name it better?)

Reply:
there are several options available: https://github.com/snoyberg/keter/blob/master/src/Keter/Config/V10.hs#L224
but I'm not sure if it's a good idea to support these.
We only use the webapp configuration at supercede because it solves a relatively hard problem of app loading:
Keep old app running, swap the app ports once the new one has finsihed booting

but the other options allow you to configure keter as an nginx replacement (with static-files or reverse-proxy), and as a background task manager.
To do that we'd need to make the rest of the stanza modifiable as well and I'm not sure if it's worth the effort.

Reply:
Can you change the `mkEnableOption` then? [^1]
```nix
    enable = lib.mkEnableOption "keter, a web app deployment manager. Note this module only supports application hotswap or whatever this is called you change this";
 ```
[^1]: Also, having two instances of your app, load balancing between them and A-B updates seems like a saner solution but that's beside the point.

Reply:
I changed the description like you suggested.
Not sure what you're getting at with loadbalancing. we don't need that, a single vm will be able to serve the entire reinsurance market.